<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-4369MQT" name="Test" tests="4" failures="1" timestamp="23 Feb 2018 18:42:34 GMT" time="249.562" errors="0">
  <testcase name="login_to_FedEx" time="22.567" classname="module1.LoginTest"/>
  <testcase name="login_to_FedEx" time="14.793" classname="module1.LoginTest"/>
  <testcase name="login_to_FedEx" time="23.421" classname="module1.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [Error] but found [Logout]">
      <![CDATA[java.lang.AssertionError: expected [Error] but found [Logout]
	at module1.LoginTest.login_to_FedEx(LoginTest.java:70)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- login_to_FedEx -->
  <testcase name="login_to_FedEx" time="40.822" classname="module1.LoginTest"/>
</testsuite> <!-- Test -->
